### PR TITLE
feat: Badge

### PR DESCRIPTION
**Pull request by manish-GH**
Preview: https://admiring-boyd-ecc99a.netlify.app/components/badge/

### CODE REVIEW

**Review by shangkaul — APPROVED** [review]
The badge is looking good, can be merged!

File: components/Avatar/styles.css [hunk 1]
```diff
@@ -8,10 +8,10 @@
 
 .avatar {
 	display: flex;
-	align-items: center;
+	align-items: flex-end;
 	justify-content: center;
-	height: 8rem;
-	width: 8rem;
+	/* height: 8rem;
+	width: 8rem; */
```
Comment (parvez-rashid) [inline components/Avatar/styles.css:14]: Remove the comment

**Review by parvez-rashid — APPROVED** [review]
Code looks good to merge.

File: components/Avatar/styles.css (hunk 1 above)
Comment (manish-GH) [reply components/Avatar/styles.css:14]: I might need this in the future, so I didn't remove it

**Pull request merged**